### PR TITLE
update client class loading server configuration

### PR DESCRIPTION
**Pull request by ryankwilliams**
## What does this implement/fix? Explain your changes

This PR updates the client class to have the same capabilities of
loading server configuration settings just like you can from the
command line. Previously it would only load server configuration
settings when instantiating the client class.

Updated example scripts to no longer need to define the default config
settings. These will be set by default.

## Does this close any currently open issues?

No open issues are related to this PR.

### CODE REVIEW

File: miqcli/api.py [hunk 1]
```diff
@@ -276,8 +276,16 @@ def __init__(self, conf, verbose=False):
         :type verbose: bool
         """
 
+        # lets first load the correct server configuration settings
+        config = Config(defaults=DEFAULT_CONFIG, verbose=verbose)
```
Comment (tvieira) [inline miqcli/api.py]: `defaults=` does not match with what it is. This is actual `settings` or something else, which, by default, receives DEFAULT_CONFIG. but I would call it something other than `defaults`. Does it make sense?

Comment (ryankwilliams) [reply miqcli/api.py]: Yes that does make sense. I would prefer `settings` over `defaults`. That parameter name has been set and I was just re-using it. Let me make an update to modify this parameter name to better map to what it actual is accepting.

**Review by vi-patel — APPROVED** [review]
looks good.

**Review by tvieira — APPROVED** [review]
LGTM

**Pull request merged**